_posts/2020-12-11-an-old-new-adventure.md

```diff
@@ -0,0 +1,61 @@
+
+Rust adoption is trending up, including at Google. The language is in good shape, but the library ecosystem is still fairly immature, missing a number of critical pieces. Building up that ecosystem is an incredibly rewarding project.
+
+I am particularly excited about Rust for font technology and infrastructure. Today, Python rules on the font design and production side, partly to the connection of typeface designer [Just van Rossum] being Guido's brother. The flexibility and expressiveness of Python makes it a good fit, but we've also gotten to a place where the *production* of fonts is done in Python, and the *consumption* is in C++.
+
+Rust lets us build reliable, performant code that can also be deployed in production, and can be the basis of fluidly interactive UI tools. I'm not the only one who sees this potential; YesLogic is building their next-generation [font shaper][Allsorts] in Rust, for many of the same reasons.
+
+The Google Fonts team has been interested in adopting more Rust for a while, and part of my role is to facilitate that. I'm really looking forward to it.
+
+## On research
+
+I have rebranded myself somewhat as a researcher, but that doesn't *quite* capture the whole story either. I have always loved research, and that love sustained the energy to complete my [PhD], but I also love building real things, and actually feel that many of these practical problems are more interesting than many of the abstract topics fashionable in academia. Just as much as writing papers and so on, I'm trying to build open source software and community around that. There isn't really a word for this role, but even without such a word I'm trying to consciously create it for myself, and am grateful that Google is allowing me to try.
+
+## On the work
+
+This is the most exciting part for me. I have a very long-term interest in 2D graphics, font technology, and UI, and have been doing a bunch of interesting things on all these fronts. I expect to spend most of my time continuing to advance research on all these frontiers.
+
+The scope of these projects is large, and more ambitious than one person could really do. That's one reason I've been consciously developing an open source community around them. That will continue.
+
+Most of the day-to-day work on [Druid] and [Runebender] will be done by Colin Rofls, though I very much enjoy getting my elbows in the code too and will be doing some of that.
+
+A major focus will be building out the [piet-gpu vision]. I believe a high-performance 2D rendering engine will be a great thing for the Rust ecosystem and with potential for large impact. It feels like good research; whether it goes into production at scale or not, I expect the things we learn from doing it will help inform the next generation of UI technology. That's equally true for research into fundamental UI principles, for example the [crochet] architecture for Druid.
```

Comment:
s/crochet/Crochet/?

Reply:
Yeah, I think we go back and forth on this. Title case for the project, lower-case for the crate.